go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/asaskevich/govalidator v0.0.0-20210307081110-f21760c49a8d
 	github.com/bakito/go-log-logr-adapter v0.0.3-0.20231211113354-bfa42fa7e121
 	github.com/cilium/ebpf v0.15.0
-	github.com/containernetworking/cni v1.1.2
+	github.com/containernetworking/cni v1.2.0
```

Comment:
Will upgrading to a newer version break our existing compatibility to existing cniVersions on the cluster?
For example, I assume we are compatible with cniVersion `1.0.0`, `2.0.0` and `3.0.0`, after this upgrading, are we still compatible?

Reply:
Yeah, it's still compatible.
Refer to https://github.com/containernetworking/cni/pull/1022, we could found that no matter the previous `skel.PluginMain` function or the latest `skel.PluginMainFuncs` all call the same `pluginMain`, which would not make it incompatible.  
It's just a supplement implementation for the latter cniVersion `1.1.0` in the future.